cli/src/test/logs.spec.ts

```diff
@@ -100,3 +101,28 @@ test('logs help', async (t) => {
   const result = await t.context.shell.run(`${t.context.run} help logs`, 'utf-8');
   t.snapshot(result);
 });
+
+test('logs by name missing app', async (t) => {
+  td.when(t.context.lycanFake.getAppByName(anything, 'not-here'))
+    .thenReject(new NotFoundError('not found'));
+
+  const result = await t.context.shell.run(`cd .. && ${t.context.run} logs not-here`, 'utf-8');
```

Comment:
Nit: `cd ..` might stay in the same directory, e.g. if we run in the root directory (of a container, presumably).  Can we `mkdir pirate && cd pirate && ...` instead?

(Also below)